.github/workflows/docs.yml

```diff
@@ -32,6 +32,6 @@ jobs:
         uses: JamesIves/github-pages-deploy-action@v4.3.0
         with:
           branch: gh-pages
-          folder: ./doc/build-docu
+          folder: ./doc/.build-docu
```

Comment:
Questions:
1. why does the process generate something inside a source folder?
2. dos the folder contain stuff that is buil**t** or something we need to buil**d** (yes, I know I am nitpicking here)